example/.gitignore

```diff
@@ -52,9 +54,10 @@ buck-out/
 # For more information about the recommended setup visit:
 # https://docs.fastlane.tools/best-practices/source-control/
 
-*/fastlane/report.xml
-*/fastlane/Preview.html
-*/fastlane/screenshots
+**/fastlane/report.xml
```

Comment:
Those 2 files are changed in `upgrade-helper`, is there a reason they are not present in your example?

![image](https://user-images.githubusercontent.com/32481228/189095622-145f5ea5-6729-418f-8f7e-6f22c4abf74d.png)

Reply:
Also these 2:
![image](https://user-images.githubusercontent.com/32481228/189095861-370622af-9be8-49ad-b115-29aa44c2d3f5.png)

Reply:
Those files weren't changed because they are not present in the example app at all.